patches/tModLoader/Terraria/DataStructures/PlayerDeathReason.cs.patch

```diff
@@ -40,6 +42,7 @@
 +	public ref int SourceProjectileType => ref _sourceProjectileType;
 +	public ref Item SourceItem => ref _sourceItem;
 +	public ref string SourceCustomReason => ref _sourceCustomReason;
++	public ref NetworkText SourceCustomReasonLocalized => ref _sourceCustomReasonLocalized;
```

Comment:
unfortunate name clash with previous member. Perhaps we should call it `CustomReason`